bin/tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+cd "`dirname $0`"
+cd ..
+
+PG_IMAGE=postgres:${PG_VERSION:-"12"}-alpine
+PG_CONTAINER_NAME=postgres
+PYTHON_VERSION=${PYTHON_VERSION:-"3.8"}
+DOCKERFILE=dev-${PYTHON_VERSION}.Dockerfile
+
+echo "starting tests for POSTGRESQL ${PG_VERSION}"
+sed "s/^FROM python:.*$/FROM python:$PYTHON_VERSION/g" dev.Dockerfile | tee $DOCKERFILE
```

Comment:
sad :(

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

> FROM instructions support variables that are declared by any ARG instructions that occur before the first FROM.

gdyby nie działało, to jest jeszcze standardowy toolik `envsubst` do tworzenia plików z templatki (korzystałem, działa)

Reply:
indeed, it works :)